packages/manager/apps/telecom/src/app/telecom/pack/move/pack-move.constant.js

```diff
@@ -72,6 +72,7 @@ const CONTEXT = {
 };
 
 const ONT_SHIPPING_CONTACT = '/me/contact/';
+const PENDING = 'pending';
```

Comment:
```suggestion
const  TASK_STATUS = { PENDING: 'pending' };
```
I think it's better to know it's the pending for what